TCAT/Utilities/SearchTableViewHelpers.swift

```diff
@@ -111,7 +119,11 @@ class SearchTableViewManager {
         
         do {
             let data = try encoder.encode(places)
-            userDefaults.set(data, forKey: key)
+            if key == Constants.UserDefaults.favorites {
```

Comment:
Is there a reason we didn't just put everything within sharedUserDefaults for simplicity?